cmd/notation/cert_test.go

```diff
@@ -1,99 +0,0 @@
-package main
```

Comment:
Any reason for not having any unit tests in this PR?

Reply:
Apart from not having the unit tests, rest all looks good to me.

Reply:
> Any reason for not having any unit tests in this PR?

Oh yeah, unit tests are in a following PR. I've also updated the comment of this PR.

Reply:
> Please post the PR link here once it is ready.

@rgnote FYI, I just added those unit tests in this PR as well. PTAL.